### ISSUE TITLE
Check for opt-in field fails if agree field contains an non-falsy unchecked value

### ISSUE THREAD

**Issue by gglnx** [verbb/formie#783]
**Description**

Integrations are triggered even if a opt-in field isn't checked. Our forms uses an agree field as a e-mail marketing integration opt-in field. The checked and unchecked values are the German words for yes (Ja) and no (Nein). Because "Nein" is not a falsy value the check in `enforceOptInField` fails and the integration is triggered without an opt-in.

As a quick fix we changed the unchecked value to "0" and the opt-in check works.

**Steps to reproduce**
1. Create a form with an agree field and use "Nein" as unchecked value
2. Enable an integration and use this agree field as opt-in field
3. Submit the form

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.10
- Craft version: 3.7.34
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.11](https://github.com/verbb/formie/releases/tag/1.5.11)
